### ISSUE TITLE
Remove empty desktops does not work when moving windows

### ISSUE THREAD

**Issue by matejdro** [kwin-scripts/kwin-tiling#197]
How to reproduce:

1. Enable "Remove empty desktops"
2.  Have at least two desktops open
3. One one of the desktops, have only one window
4. Move last window to another desktop (for example using "Window One Desktop to the left" KWin shortcut)
5. That desktop will not be removed even though it is empty.

Is there a way to move a window to another desktop through this script that I'm missing?

**Comment by faho** [comment]
Yes, it only checks if a desktop is empty when a client on it is removed.
